classroom.rb

```diff
@@ -0,0 +1,42 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    @students << student
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Great job so far in your implementation! 🚀 However, it seems that your `add_rental` method might not be working as expected. The `add_rental` method should take two parameters, `date` and `person`. Additionally, the book itself should also be passed as a parameter to create the new rental, achieving the 'has-many' association. 😊

To achieve this, consider implementing your method like this:

```ruby
def rent_book(person, date)
  Rental.new(date, person, self)
end
```

This revised approach will ensure that the rental is associated with the book correctly. 
Happy coding! 😊🚀👨‍💻